_pytest/test_emulators.py

```diff
@@ -38,7 +38,7 @@ def test_wit_request():
 def test_wit_response():
     from rasa_nlu.emulators.wit import WitEmulator
     em = WitEmulator()
-    data = {"text": "I want italian food", "intent": "inform", "entities": {"cuisine": "italian"}}
+    data = {"text": "I want italian food", "intent": "inform", "entities": [{"entity": "cuisine", "value": "italian"}]}
```

Comment:
I checked how wit formats the response and those keys are not present in the original one. Do you want me to add them anyway?